crates/bevy_core_pipeline/src/prepass/mod.rs

```diff
@@ -0,0 +1,144 @@
+//! Run a prepass before the main pass to generate depth and/or normals texture, sometimes called a thin g-buffer.
```

Comment:
I would add a small section somewhere about how the depth prepass always runs and reduces overdraw if normals (or in the future velocity) are enabled, and the component just controls whether it's available or not.